.github/workflows/test.yaml

```diff
@@ -55,3 +55,6 @@ jobs:
       flavor: ${{ matrix.flavor }}
       type: ${{ matrix.type }}
     secrets: inherit # Inherits all secrets from the parent workflow.
+
+  test-security-hub-scan:
```

Comment:
just a note on having this in the end to end test workflow - this is probably fine for now but it may be beneficial to think about splitting it - especially if you wanted to use in-package SBOMs in the future (the tests currently exclude those to save time and SBOMs are only generated on release) - the can results are somewhat useful on PRs for tracking though I would argue would be more so for customers on release and/or in a cron job workflow that runs more consistently (which could be a separate workflow that specifies a timeframe to run within)

Reply:
I added this here because I only saw this one place for testing, but I'd be happy to split that out if that's preferable.

Reply:
Thinking on this more I do think it would be good to split out into it's own workflow - this also does not matrix flavors so we will only get one result from it.